jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -166,6 +174,7 @@
            - raw
            - rbd
            - netapp
+           - nfs
```

Comment:
I think I'll rework that once more. And put that check into mklcloud's sanity checks.